### ISSUE TITLE
Rebase test playlist webui `components/playlist/resources` and others

### ISSUE THREAD

**Issue by sangwoo108** [brave/brave-browser#24027]

**Comment by simonhong** [comment]
It would be nice to have test webui like brave://playlist.
I think putting it as a sidebar item would be great.

**Comment by sangwoo108** [comment]
> It would be nice to have test webui like brave://playlist.
>  think putting it as a sidebar item would be great.

Yeah. That's cool. I'll try attach a button on the sidebar on the current PR.

**Comment by simonhong** [comment]
> > It would be nice to have test webui like brave://playlist.
> > think putting it as a sidebar item would be great.
> 
> Yeah. That's cool. I'll try attaching a button on the sidebar on the current PR.

Let's do it after @petemill completes sidepanel integration to sidebar (https://github.com/brave/brave-core/pull/13769).
It has many sidebar impls changes. If you do that now, it'll make many conflicts.
